> Initialized logger
> 127.0.0.1 - - [12/Mar/2014 12:27:55] "GET / HTTP/1.1" 200 -
> 127.0.0.1 - - [12/Mar/2014 12:27:56] "GET /static/css/bootstrap.min.css HTTP/1.1" 200 -
> 127.0.0.1 - - [12/Mar/2014 12:27:56] "GET /static/css/bootstrap-responsive.css HTTP/1.1" 200 -
> 127.0.0.1 - - [12/Mar/2014 12:27:56] "GET /static/js/sha1.js HTTP/1.1" 200 -
> 127.0.0.1 - - [12/Mar/2014 12:27:56] "GET /static/js/jquery.js HTTP/1.1" 200 -
> 127.0.0.1 - - [12/Mar/2014 12:27:56] "GET /static/js/bootstrap.min.js HTTP/1.1" 200 -
> 127.0.0.1 - - [12/Mar/2014 12:28:00] "POST /login/ HTTP/1.1" 302 -
> 127.0.0.1 - - [12/Mar/2014 12:28:00] "GET /monitoring HTTP/1.1" 301 -
> 127.0.0.1 - - [12/Mar/2014 12:28:00] "GET /monitoring/ HTTP/1.1" 500 -
> Traceback (most recent call last):
>   File "C:\Python27\lib\site-packages\flask\app.py", line 1836, in __call__
>     return self.wsgi_app(environ, start_response)
>   File "C:\Python27\lib\site-packages\flask\app.py", line 1820, in wsgi_app
>     response = self.make_response(self.handle_exception(e))
>   File "C:\Python27\lib\site-packages\flask\app.py", line 1403, in handle_exception
>     reraise(exc_type, exc_value, tb)
>   File "C:\Python27\lib\site-packages\flask\app.py", line 1817, in wsgi_app
>     response = self.full_dispatch_request()
>   File "C:\Python27\lib\site-packages\flask\app.py", line 1477, in full_dispatch_request
>     rv = self.handle_user_exception(e)
>   File "C:\Python27\lib\site-packages\flask\app.py", line 1381, in handle_user_exception
>     reraise(exc_type, exc_value, tb)
>   File "C:\Python27\lib\site-packages\flask\app.py", line 1475, in full_dispatch_request
>     rv = self.dispatch_request()
>   File "C:\Python27\lib\site-packages\flask\app.py", line 1461, in dispatch_request
>     return self.view_functions[rule.endpoint](**req.view_args)
>   File "C:\Python27\lib\site-packages\flask_login.py", line 732, in decorated_view
>     return func(*args, **kwargs)
>   File "C:\Users\RSK\Documents\GitHub\Malaria-Server\serveus\views.py", line 56, in returned
>     return function(*args, **kwargs)
>   File "C:\Users\RSK\Documents\GitHub\Malaria-Server\serveus\views.py", line 302, in monitoring
>     for i in unique_coor:
>   File "C:\Python27\lib\site-packages\sqlalchemy\orm\query.py", line 2391, in __iter__
>     return self._execute_and_instances(context)
>   File "C:\Python27\lib\site-packages\sqlalchemy\orm\query.py", line 2406, in _execute_and_instances
>     result = conn.execute(querycontext.statement, self._params)
>   File "C:\Python27\lib\site-packages\sqlalchemy\engine\base.py", line 717, in execute
>     return meth(self, multiparams, params)
>   File "C:\Python27\lib\site-packages\sqlalchemy\sql\elements.py", line 317, in _execute_on_connection
>     return connection._execute_clauseelement(self, multiparams, params)
>   File "C:\Python27\lib\site-packages\sqlalchemy\engine\base.py", line 814, in _execute_clauseelement
>     compiled_sql, distilled_params
>   File "C:\Python27\lib\site-packages\sqlalchemy\engine\base.py", line 927, in _execute_context
>     context)
>   File "C:\Python27\lib\site-packages\sqlalchemy\engine\base.py", line 1076, in _handle_dbapi_exception
>     exc_info
>   File "C:\Python27\lib\site-packages\sqlalchemy\util\compat.py", line 185, in raise_from_cause
>     reraise(type(exception), exception, tb=exc_tb)
>   File "C:\Python27\lib\site-packages\sqlalchemy\engine\base.py", line 920, in _execute_context
>     context)
>   File "C:\Python27\lib\site-packages\sqlalchemy\engine\default.py", line 425, in do_execute
>     cursor.execute(statement, parameters)
> OperationalError: (OperationalError) no such column: case.chunklist_id u'SELECT "case".id AS case_id, "case".date AS case_date, "case".parasite AS case_parasite, "case".description AS case_description, "case".comment AS case_comment, "case".lat AS case_lat, "case".lng AS case_lng, "case".user_id AS case_user_id, "case".test AS case_test, "case".partype_id AS case_partype_id, "case".region_id AS case_region_id, "case".province_id AS case_province_id, "case".municipality_id AS case_municipality_id, "case".chunklist_id AS case_chunklist_id, "case".parasite_validator AS case_parasite_validator, "case".description_validator AS case_description_validator \nFROM "case" GROUP BY "case".lat, "case".lng' ()
> 127.0.0.1 - - [12/Mar/2014 12:28:00] "GET /monitoring/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
> 127.0.0.1 - - [12/Mar/2014 12:28:00] "GET /monitoring/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
> 127.0.0.1 - - [12/Mar/2014 12:28:00] "GET /monitoring/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
> 127.0.0.1 - - [12/Mar/2014 12:28:00] "GET /monitoring/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
> 127.0.0.1 - - [12/Mar/2014 12:28:00] "GET /monitoring/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
> 127.0.0.1 - - [12/Mar/2014 12:28:00] "GET /monitoring/?__debugger__=yes&cmd=resource&f=source.png HTTP/1.1" 200 -
> 127.0.0.1 - - [12/Mar/2014 12:28:00] "GET /monitoring/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
